server/routes/api/case.js

```diff
@@ -74,6 +87,8 @@ router.put('/:caseId/note/:noteId',
 
 router.get('/:caseId/summary', caseSummaryMiddleware, caseSummaryApiResponseMiddleware);
 
+router.get('/:caseId/actions', caseActionDataMiddleware, caseActionApiResponseMiddleware);
```

Comment:
This single caseActionDataMiddleware function is used for both gets and posts and seems to be quite complex. It looks like it might have multiple responsibilities. Would we get simpler methods if we split it up?

Reply:
With both the `GET` and `POST` requests the role of this middleware is the same thing, i.e. to get the current info. So to separate it in this instance I think would only lead to more duplicate code.